e2e/test/iothub/method/MethodFaultInjectionTests.cs

```diff
@@ -22,7 +22,7 @@ namespace Microsoft.Azure.Devices.E2ETests.Methods
     [TestCategory("FaultInjection")]
     public class MethodFaultInjectionTests : E2EMsTestBase
     {
-        private readonly string DevicePrefix = $"E2E_{nameof(MethodFaultInjectionTests)}_";
+        private readonly string DevicePrefix = $"{nameof(MethodFaultInjectionTests)}_";
```

Comment:
Can you provide a bit of context on why we decided to make this change? I am just curious.

Reply:
Not sure actually. @abhipsaMisra ?

Reply:
The "E2E_" prefix was modified to be a part of `TestDevice` directly. 

This change came as a part of the "twin.modelId" commit that we brought in from preview, afaik, but @vinagesh can help confirm that!